.github/workflows/sync_from_airtable.yml

```diff
@@ -0,0 +1,344 @@
+                                return true;
+                            }
+                        })
+                    }
+
+                    return acc;
+                }, { keepAssignees: new Array(issue.data.assignees.length), removeAssignees: [], addAssignees: [], priorAssignees: issue.data.assignees.map((assignee) => assignee.login)});
+
+                if (addAssignees.length > 0) {
+                    await github.rest.issues.addAssignees({
+                        owner: OWNER,
+                        repo: REPO,
+                        issue_number: NUMBER,
+                        assignees: addAssignees
+                    });
+                    core.info('Assignees Added: ', addAssignees.join(','))
+                }
+                if (removeAssignees.length > 0) {
+                    await github.rest.issues.removeAssignees({
+                        owner: OWNER,
+                        repo: REPO,
+                        issue_number: NUMBER,
+                        assignees: removeAssignees
+                    });
+                    core.info('Assignees Removed: ', removeAssignees.join(','))
+                }
+            }
+
+  labeled_unlabeled:
+    name: Labeled or Unlabeled
+    needs: existing_issue
+    if: (fromJSON(needs.existing_issue.outputs.github_issue).status == 200 ) && (github.event.client_payload.action == 'labeled/unlabeled')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        env:
+          DATA: ${{ fromJSON(env.DATA) }}
+          ISSUE: ${{ needs.existing_issue.outputs.github_issue }}
+          NUMBER: ${{env.NUMBER}}
+          OWNER: ${{env.OWNER}}
+          REPO: ${{env.REPO}}
+        with:
+          script: |
+            const { DATA, NUMBER, ISSUE, OWNER, REPO } = process.env;
+            const { labels } = JSON.parse(DATA);
+            if (labels) {
+                const issue  = JSON.parse(process.env.ISSUE);
+                const { removeLabels, addLabels } = labels.reduce((acc, labelName, idx) => {
+                    const currentIdx = acc.priorLabels.indexOf(labelName);
+
+                    if (currentIdx === -1) {
+                        acc.addLabels.push(labelName);
+                    } else {
+                        // Keep the label indices from the priorLabel list
+                        acc.keepLabels[currentIdx] = labelName;
+                    }
+
+                    if (idx === labels.length - 1) {
+                        // We are in our final loop. Remove empty indices from keepLabels and set those priorLabels for removal.
+                        acc.keepLabels = acc.keepLabels.filter((label, keepLabelIdx) => {
+                            if (label === undefined) {
+                                acc.removeLabels.push(acc.priorLabels[keepLabelIdx]);
+                                return false;
+                            } else {
+                                return true;
```

Comment:
Nitpick: `return true` doesn't need to be in an else statement because the if block returns